pkg/pillar/cmd/zedagent/handlecontent.go

```diff
@@ -59,7 +85,7 @@ func parseContentInfoConfig(ctx *getconfigContext,
 	for _, cfgContentTree := range cfgContentTreeList {
 		contentConfig := new(types.ContentTreeConfig)
 		contentConfig.ContentID, _ = uuid.FromString(cfgContentTree.GetUuid())
-		contentConfig.DatastoreID, _ = uuid.FromString(cfgContentTree.GetDsId())
+		contentConfig.DatastoreIDsList, _ = getDatastoreIDsList(cfgContentTree)
```

Comment:
I followed the previous semantics - ignoring the error. But of course can add those output to the getDatastoreIDsList.

Reply:
It looks like uuid.FromString() returns object, not a reference, thus the previous semantic here is to return uuid.UUID{} (nilUUID) in case of omitted error.

Reply:
This (having a 1 element as nil uuid in case of an error) I already fixed locally, no objections. Mainly my comment was related to your "not safe to ignore error", but rereading it once again I understand that you meant it in a combination with "and use DatastoreIDsList[0] later". So all good, all fixed :)